src/edit/main.js

```diff
@@ -156,21 +156,22 @@ class ProseMirror {
       domDrop: new DOMSubscription
     }
 
-    if (opts.place && opts.place.appendChild) {
-      this.root = opts.place
+    if (opts.place && opts.place.appendChild)
+      opts.place.appendChild(this.wrapper)
+    else if (opts.place)
+      opts.place(this.wrapper)
+
+    // does not support opts.place being a function and asynchronously mount prosemirror
+    if (opts.place) {
```

Comment:
If `opts.place` *does* asynchronously place the wrapper (i.e. `this.wrapper.parentNode == null`) then won't the code that follows throw an exception?